Server/ConcordiaCurriculumManagerTest/UnitTests/Controller/GroupControllerTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿using ConcordiaCurriculumManager.Controllers;
+using ConcordiaCurriculumManager.Services;
+using ConcordiaCurriculumManager.Models.Users;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Moq;
+using System.Net;
+using ConcordiaCurriculumManager.DTO;
+
+namespace ConcordiaCurriculumManagerTest.UnitTests.Controllers
+{
+    [TestClass]
+    public class GroupControllerTests
+    {
+        private Mock<IGroupService> _groupService = null;
```

Comment:
For reference, you can remove these warnings by, instead of `= null;`, you change it to `= null!;`. This is the null forgiving operator (basically saying yes I know there's a potential issue but I don't care).